dist/cli.js

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env node
```

Comment:
@harterSe Correct me if im wrong, but the compiled code should be part of the package i think. So it can be used in regular JS

Reply:
It would be possible to publish the dist folder to npm.

Then all users that use the package with `npm i svg-chameleon` will install the dist with js files and .d.ts files for use in js and typsescript projects.
And users that checkout the git project can build the source.